Sources/Plasma/FeatureLib/pfPatcher/plManifests.cpp

```diff
@@ -52,24 +52,44 @@ Mead, WA   99021
 #   define MANIFEST(in, ex) in
 #endif // PLASMA_EXTERNAL_RELEASE
 
+#if HS_BUILD_FOR_APPLE
+#   define PL_EXECUTABLE_SUFFIX ".app"
```

Comment:
The executable suffix is not really Plasma specific, so I don't think the `PL_` suffix is appropriate here.

Reply:
Bump